packages/get-starknet/src/rpcs/deployment-data.test.ts

```diff
@@ -0,0 +1,25 @@
+import { mockWalletInit, createWallet } from '../__tests__/helper';
+import { MetaMaskSnap } from '../snap';
+import type { DeploymentData } from '../type';
+import { WalletDeploymentData } from './deployment-data';
+
+describe('WalletDeploymentData', () => {
+  it('returns deployment data', async () => {
```

Comment:
shall we test that the right error is triggered in case account is deployed ?

Reply:
it is the responsibility of the snap, we trust the snap response

Reply:
and there is a test to test the error from snap convert to correct response, so no need to implement here

Reply:
ok!